src/DotNetBoilerplate.Infrastructure/DAL/Migrations/20240729185816_Add-Reviews.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace DotNetBoilerplate.Infrastructure.DAL.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddReviews : Migration
```

Comment:
Pusta migracja - coś jest nie tak. Jak jest problem z tym to dawajcie znać i wspólnie naprawimy